WARN  2015-06-14 14:24:08,471 (NullSarosSession.java:64) unexpected method call
WARN  2015-06-14 14:24:08,565 (NullSarosSession.java:64) unexpected method call
WARN  2015-06-14 14:24:50,405 (NullSarosSession.java:64) unexpected method call
WARN  2015-06-14 14:24:50,465 (NullSarosSession.java:64) unexpected method call
ERROR 2015-06-14 14:26:04,167 (UPnPServiceImpl.java:146) creating port mapping failed: port is already mapped by another application: null
java.io.IOException: port is already mapped by another application: null
	at de.fu_berlin.inf.dpp.net.upnp.internal.UPnPServiceImpl.createPortMapping(UPnPServiceImpl.java:102)
	at de.fu_berlin.inf.dpp.net.SarosNet$4.run(SarosNet.java:494)
	at de.fu_berlin.inf.dpp.util.Utils$2.run(Utils.java:175)
	at java.lang.Thread.run(Unknown Source)
WARN  2015-06-14 14:26:04,170 (SarosNet.java:498) failed to create port mapping on device: VEGN2610 (Gateway) [7777|TCP]
ERROR 2015-06-14 14:26:04,539 (SarosNet.java:385)  new connection state == ERROR
SASL authentication DIGEST-MD5 failed: not-authorized: 
	at org.jivesoftware.smack.SASLAuthentication.authenticate(SASLAuthentication.java:342)
	at org.jivesoftware.smack.XMPPConnection.login(XMPPConnection.java:243)
	at de.fu_berlin.inf.dpp.net.SarosNet.connect(SarosNet.java:261)
	at de.fu_berlin.inf.dpp.Saros.connect(Saros.java:660)
	at de.fu_berlin.inf.dpp.Saros$2.run(Saros.java:600)
	at de.fu_berlin.inf.dpp.util.Utils$2.run(Utils.java:175)
	at java.lang.Thread.run(Unknown Source)
ERROR 2015-06-14 14:26:06,559 (SarosNet.java:385)  new connection state == ERROR
SASL authentication DIGEST-MD5 failed: not-authorized: 
	at org.jivesoftware.smack.SASLAuthentication.authenticate(SASLAuthentication.java:342)
	at org.jivesoftware.smack.XMPPConnection.login(XMPPConnection.java:243)
	at de.fu_berlin.inf.dpp.net.SarosNet.connect(SarosNet.java:261)
	at de.fu_berlin.inf.dpp.Saros.connect(Saros.java:660)
	at de.fu_berlin.inf.dpp.ui.actions.ChangeXMPPAccountAction$6.run(ChangeXMPPAccountAction.java:195)
	at de.fu_berlin.inf.dpp.util.Utils$2.run(Utils.java:175)
	at java.lang.Thread.run(Unknown Source)
ERROR 2015-06-14 14:26:08,725 (UPnPServiceImpl.java:146) creating port mapping failed: port is already mapped by another application: null
java.io.IOException: port is already mapped by another application: null
	at de.fu_berlin.inf.dpp.net.upnp.internal.UPnPServiceImpl.createPortMapping(UPnPServiceImpl.java:102)
	at de.fu_berlin.inf.dpp.net.SarosNet$4.run(SarosNet.java:494)
	at de.fu_berlin.inf.dpp.util.Utils$2.run(Utils.java:175)
	at java.lang.Thread.run(Unknown Source)
WARN  2015-06-14 14:26:08,726 (SarosNet.java:498) failed to create port mapping on device: VEGN2610 (Gateway) [7777|TCP]
WARN  2015-06-14 14:26:13,298 (StunServiceImpl.java:420) received no response from STUN server stunserver.org/69.172.201.208:3478 at local address /10.0.0.169:0
WARN  2015-06-14 14:26:13,298 (StunServiceImpl.java:420) received no response from STUN server stunserver.org/69.172.201.208:3478 at local address /169.254.207.181:0
WARN  2015-06-14 14:26:16,359 (StunServiceImpl.java:420) received no response from STUN server stunserver.org/69.172.201.208:3478 at local address /169.254.207.181:0
WARN  2015-06-14 14:26:16,359 (StunServiceImpl.java:420) received no response from STUN server stunserver.org/69.172.201.208:3478 at local address /10.0.0.169:0
ERROR 2015-06-14 14:26:25,520 (UPnPServiceImpl.java:146) creating port mapping failed: port is already mapped by another application: null
java.io.IOException: port is already mapped by another application: null
	at de.fu_berlin.inf.dpp.net.upnp.internal.UPnPServiceImpl.createPortMapping(UPnPServiceImpl.java:102)
	at de.fu_berlin.inf.dpp.net.SarosNet$4.run(SarosNet.java:494)
	at de.fu_berlin.inf.dpp.util.Utils$2.run(Utils.java:175)
	at java.lang.Thread.run(Unknown Source)
WARN  2015-06-14 14:26:25,522 (SarosNet.java:498) failed to create port mapping on device: VEGN2610 (Gateway) [7777|TCP]
WARN  2015-06-14 14:26:33,118 (StunServiceImpl.java:420) received no response from STUN server stunserver.org/69.172.201.208:3478 at local address /169.254.207.181:0
WARN  2015-06-14 14:26:33,118 (StunServiceImpl.java:420) received no response from STUN server stunserver.org/69.172.201.208:3478 at local address /10.0.0.169:0
WARN  2015-06-14 14:56:03,386 (NullSarosSession.java:64) unexpected method call
WARN  2015-06-14 14:56:03,389 (NullSarosSession.java:64) unexpected method call
WARN  2015-06-14 14:56:03,821 (SWTUtils.java:144) could not execute runnable de.fu_berlin.inf.dpp.ui.widgets.ConnectionStateComposite$1$1@6bc90642, UI thread is not available
StackTrace:
	at de.fu_berlin.inf.dpp.ui.util.SWTUtils.runSafeSWTAsync(SWTUtils.java:145)
	at de.fu_berlin.inf.dpp.ui.widgets.ConnectionStateComposite$1.connectionStateChanged(ConnectionStateComposite.java:55)
	at de.fu_berlin.inf.dpp.net.SarosNet.setConnectionState(SarosNet.java:390)
	at de.fu_berlin.inf.dpp.net.SarosNet.disconnect(SarosNet.java:289)
	at de.fu_berlin.inf.dpp.Saros$1.run(Saros.java:334)
	at de.fu_berlin.inf.dpp.util.Utils$2.run(Utils.java:175)
	at java.lang.Thread.run(Unknown Source)
WARN  2015-06-14 14:56:03,981 (SWTUtils.java:144) could not execute runnable de.fu_berlin.inf.dpp.ui.widgets.ConnectionStateComposite$1$1@1b142585, UI thread is not available
StackTrace:
	at de.fu_berlin.inf.dpp.ui.util.SWTUtils.runSafeSWTAsync(SWTUtils.java:145)
	at de.fu_berlin.inf.dpp.ui.widgets.ConnectionStateComposite$1.connectionStateChanged(ConnectionStateComposite.java:55)
	at de.fu_berlin.inf.dpp.net.SarosNet.setConnectionState(SarosNet.java:390)
	at de.fu_berlin.inf.dpp.net.SarosNet.disconnect(SarosNet.java:293)
	at de.fu_berlin.inf.dpp.Saros$1.run(Saros.java:334)
	at de.fu_berlin.inf.dpp.util.Utils$2.run(Utils.java:175)
	at java.lang.Thread.run(Unknown Source)
